src/Nri/Ui/QuestionBox/V3.elm

```diff
@@ -298,15 +311,37 @@ viewGuidance config referencingId markdown_ =
                     , Css.position Css.relative
                     ]
                 ]
-                [ viewCharacter character_
-                , viewSpeechBubble config
-                    referencingId
-                    [ Balloon.markdown markdown_
-                    , Balloon.onLeft
-                    , Balloon.alignArrowEnd
-                    , Balloon.css [ Css.minHeight (Css.px 46) ]
-                    ]
-                ]
+                ([ config.textToSpeechView
+                    |> Maybe.map
+                        (div
+                            [ css
+                                [ Css.position Css.relative
+                                , Css.zIndex (Css.int 1)
+                                , Css.left (Css.px -24)
+                                , Css.top (Css.px 8)
+                                , withMedia [ quizEngineMobile ]
+                                    [ Css.left Css.auto
+                                    , Css.top Css.auto
+                                    , Css.float Css.left
+                                    , Css.padding4 Css.zero (Css.px 5) Css.zero Css.zero
+                                    , Css.position Css.static
+                                    ]
+                                ]
+                            ]
+                            << List.singleton
+                        )
+                 , Just <| viewCharacter character_
+                 , Just <|
+                    viewSpeechBubble config
+                        referencingId
+                        [ Balloon.markdown markdown_
+                        , Balloon.onLeft
+                        , Balloon.alignArrowEnd
+                        , Balloon.css [ Css.minHeight (Css.px 46) ]
+                        ]
+                 ]
+                    |> List.filterMap identity
```

Comment:
I think changing the modeling from `textToSpeechView : Maybe (Html msg)` to `textToSpeechView: Html msg`, initialized as `textToSpeechView = text ""` might simplify the view code a bit.

Currently, there's actually a bug in that code that I think becomes a bit more obvious if there's no need for the List.filterMap logic:

https://user-images.githubusercontent.com/8811312/215192259-fd3bc83e-a331-4abf-ad53-b7931b9af477.mov